napalm_eos/eos.py

```diff
@@ -85,6 +85,9 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
 
         self.enablepwd = optional_args.get('enable_password', '')
 
+    def __del__(self):
```

Comment:
I think I missed this on a previous PR. Why this? If we want this functionality, this should be in `napalm-base` in any case.